examples/interbot/bots/master/actions/delegate_to_bots.js

```diff
@@ -0,0 +1,63 @@
+const axios = require('axios')
+const _ = require('lodash')
+
+/**
+ * Tries to delegate the question to one of the specified sub-bots
+ *
+ * The state will be amended as follow..
+ *
+ * **No delegation found**
+ * `temp.delegation` => `[]`
+ *
+ * **Delegation(s) found**
+ * `temp.delegation` => `[ { botId: '', confidence: 0.5, botUrl: '', answer: '' } ]`
+ * > Results will be sorted by confidence ranking
+ *
+ * @title Get Delegation
+ * @category Delegation
+ * @param {string} [bots=sub1,sub2] The botId of the bots to ask (case-sensitive). Split with a comma.
+ * @param {Number} [minConfidence=0.5] The minimum NLU confidence of the sub-bot to be considered a match
+ */
+
+const exec = async (bots = '', minConfidence = '0.5') => {
+  const random = Math.random()
+    .toString()
+    .substr(3, 8)
+  const uniqueUserId = `delegate_${event.target}_${random}`
+
+  minConfidence = Number(minConfidence)
+  minConfidence = isNaN(minConfidence) ? 0.5 : minConfidence
+
+  const botIds = bots
+    .split(',')
+    .map(x => x.trim())
+    .filter(x => x.length)
+
+  const matches = []
+
+  for (let botId of botIds) {
```

Comment:
i'd add a try catch for the axios output, so errors are correctly handled

Reply:
done